### PR TITLE
Website and repo name updates (+ lowercase Supermarq/Supercheq) 

### PR DESCRIPTION
**Pull request by bharat-thotakura**
Changes include: 

- Updated website link to [infleqtion.com/superstaq](www.infleqtion.com/superstaq)
- Updated GitHub naming from SupertechLabs -> Infleqtion,
- Updated Super.tech -> Superstaq development team, 
- Email updates to superstaq@infleqtion.com, 
- Updated SupermarQ -> Supermarq,
- Updated SupercheQ -> Supercheq. 

Completes docs side of #531.

### CODE REVIEW

**Comment by knsmith** [comment]
working on reviewing this now!

**Comment by knsmith** [comment]
small nit, but perhaps we change 
`client-superstaq/supermarq-benchmarks/examples/SupermarQ_HPCA_Tutorial.ipynb`
to
`client-superstaq/supermarq-benchmarks/examples/Supermarq_HPCA_Tutorial.ipynb`

**Comment by knsmith** [comment]
Found a few more instances of "SuperstaQ " (omitting cases like SuperstaQProvider:

Line 13 in `general-superstaq/general_superstaq/superstaq_client.py`

`
"""Client for making requests to SuperstaQ's API."""
`

Line 798 in `general-superstaq/general_superstaq/superstaq_client_test.py`

`
    with pytest.raises(EnvironmentError, match="SuperstaQ API key not specified and not found."):
`

Line 21 in `qiskit-superstaq/qiskit_superstaq/superstaq_provider_test.py`

`
    with pytest.raises(EnvironmentError, match="SuperstaQ API key not specified and not found."):
`

**Comment by knsmith** [comment]
I might be suggesting changes out of focus for this PR (exceptions), but this info be used for other follow up PRs...

**Comment by bharat-thotakura** [comment]
> Found a few more instances of "SuperstaQ " (omitting cases like SuperstaQProvider):
> 
> Line 13 in `general-superstaq/general_superstaq/superstaq_client.py`
> 
> `"""Client for making requests to SuperstaQ's API."""`
> 
> Line 798 in `general-superstaq/general_superstaq/superstaq_client_test.py`
> 
> `with pytest.raises(EnvironmentError, match="SuperstaQ API key not specified and not found."):`
> 
> Line 21 in `qiskit-superstaq/qiskit_superstaq/superstaq_provider_test.py`
> 
> `with pytest.raises(EnvironmentError, match="SuperstaQ API key not specified and not found."):`

Thanks, yeah gss is the only one that still needs to be updated. It'll be a separate PR related to [#2375](https://github.com/SupertechLabs/server-superstaq/issues/2375)

**Comment by knsmith** [comment]
Last major comment post lunch discussion: Super.tech -> Superstaq development team within code/docs

**Comment by bharat-thotakura** [comment]
> Last major comment post lunch discussion: Super.tech -> Superstaq development team within code/docs

I've updated all client instances of Super.tech to Superstaq development team, though not sure if the full phrase is needed for all instances.

**Comment by bharat-thotakura** [comment]
Also, just to double-check, should the author's email be updated from pranav@super.tech -> pranav.gokhale@coldquanta.com?

**Comment by knsmith** [comment]
> Also, just to double-check, should the author's email be updated from [pranav@super.tech](mailto:pranav@super.tech) -> [pranav.gokhale@coldquanta.com](mailto:pranav.gokhale@coldquanta.com)?

I thought we wanted to go with `superstaq@infleqtion.com` with the author being 'Superstaq development team', but let's confirm with @singular-value . 

Going in this direction seems to agree with what Cirq has in their setup.py:
image

**Comment by knsmith** [comment]
> > Last major comment post lunch discussion: Super.tech -> Superstaq development team within code/docs
> 
> I've updated all client instances of Super.tech to Superstaq development team, though not sure if the full phrase is needed for all instances.

Are there any particular examples where it doesn't seem like the best choice?

**Comment by bharat-thotakura** [comment]
^ Oh yeah I think it fits pretty much, just wondering about other's opinions on minor instances such as:

In `README.md`: "This repository is the home of Superstaq development team's ..." to just "This repository is the home of Superstaq ..."?

Likewise in `general-superstaq/README.md`: "... code shared across Superstaq development team's codebase ..." -> "... code shared across Infleqtion's codebase ..."?

In `qiskit-superstaq/examples/uchicago_workshop.ipynb`: "Superstaq development team focuses on quantum benchmarking ..." ->  "Superstaq focuses on ... "  or "Infleqtion's software org ... "? for example.

**Comment by bharat-thotakura** [comment]
> > Also, just to double-check, should the author's email be updated from [pranav@super.tech](mailto:pranav@super.tech) -> [pranav.gokhale@coldquanta.com](mailto:pranav.gokhale@coldquanta.com)?
> 
> I thought we wanted to go with `superstaq@infleqtion.com` with the author being 'Superstaq development team', but let's confirm with @singular-value .
> 
> Going in this direction seems to agree with what Cirq has in their setup.py: image

Ah right, just wanted to double-check the exact email name.

**Comment by singular-value** [comment]
Confirmed (superstaq@infleqtion.com)

**Comment by knsmith** [comment]
> ^ Oh yeah I think it fits pretty much, just wondering about other's opinions on minor instances such as:
> 
> In `README.md`: "This repository is the home of Superstaq development team's ..." to just "This repository is the home of Superstaq ..."?
> 
> Likewise in `general-superstaq/README.md`: "... code shared across Superstaq development team's codebase ..." -> "... code shared across Infleqtion's codebase ..."?
> 
> In `qiskit-superstaq/examples/uchicago_workshop.ipynb`: "Superstaq development team focuses on quantum benchmarking ..." -> "Superstaq focuses on ... " or "Infleqtion's software org ... "? for example.

This is fair...I'd say go with your gut for whatever sounds the best / more natural. 

This is looking good! Going to check over again so we can try to get this approved ASAP.

**Comment by bharat-thotakura** [comment]
^ Thanks, yeah I think I'll leave it as is for consistent naming, but maybe future PRs can update the specific wording instances (if need be) on a case-by-case basis.

**Review by knsmith — APPROVED** [review]
LGTM!

File: supermarq-benchmarks/setup.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 setup(
     name=name,
     version=__version__,
-    url="https://github.com/SupertechLabs/SupermarQ",
+    url="https://github.com/Infleqtion/client-superstaq",
     author="Super.tech",
     author_email="pranav@super.tech",
```
[the remark this answers is not in the document]
Comment (bharat-thotakura) [reply supermarq-benchmarks/setup.py]: Done

File: general-superstaq/setup.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 setup(
     name=name,
     version=__version__,
-    url="https://github.com/SupertechLabs/general-superstaq",
+    url="https://github.com/Infleqtion/general-superstaq",
     author="Super.tech",
     author_email="pranav@super.tech",
```
[the remark this answers is not in the document]
Comment (bharat-thotakura) [reply general-superstaq/setup.py]: Done

File: cirq-superstaq/setup.py [hunk 3]
```diff
@@ -36,7 +36,7 @@
 setup(
     name=name,
     version=__version__,
-    url="https://github.com/SupertechLabs/cirq-superstaq",
+    url="https://github.com/Infleqtion/cirq-superstaq",
     author="Super.tech",
     author_email="pranav@super.tech",
```
[the remark this answers is not in the document]
Comment (bharat-thotakura) [reply cirq-superstaq/setup.py]: Done

**Comment by bharat-thotakura** [comment]
Thanks @knsmith! Will merge if there aren't any further suggestions from others. 

cc: @vtomole @richrines1 @dowusu-antwi regarding redirects to client-superstaq and updates to setup.

**Pull request merged**